src/commit_templater.rs

```diff
@@ -248,6 +248,10 @@ fn build_commit_keyword_opt<'repo>(
             let maybe_entries = repo.resolve_change_id(commit.change_id());
             maybe_entries.map_or(0, |entries| entries.len()) > 1
         })),
+        "obsolete" => language.wrap_boolean(wrap_fn(property, |commit| {
```

Comment:
The keyword should also be renamed to "hidden" (or "visible" if we prefer a positive term)?

I think "hidden" is technically correct than "obsolete".

Reply:
I was using "obsolete" with the thought that, eventually, these can be garbage collected.

In my mind, a hidden commit is one that's not shown by my default revset (so, most commits on the main branch).

But it seems people don't share this sentiment. If I don't hear anything, I will change it to "visible" or "hidden".

Reply:
Maybe I'm too used to using Mercurial, but I think "obsolete" is a commit superseded by something else (or abandoned), which could be temporarily un-"hidden" by visible descendants (causing divergence.)

> a hidden commit is one that's not shown by my default revset

That's an interesting point. Maybe "hidden" is too weak to describe a permanent property? Show/hide is a common word in tree-like UI to temporarily switch the visibility.

Reply:
> That's exactly what these are, as far as I can tell.

Well, I meant anything other than the latest entry in obslog are "obsolete", but not always "hidden". Perhaps, my "hidden" definition is identical to your "obsolete".

Reply:
I think we should go with "hidden" for now, mostly because it's consistent with `visible_heads()` (and `unobsolete_heads()` or `valid_heads()` doesn't sounds great either).

Reply:
OK, let's go with hidden.